### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by anders-kiaer**

### CODE REVIEW

File: webviz_subsurface/plugins/_well_cross_section_fmu.py [hunk 1]
```diff
@@ -118,25 +135,26 @@ def __init__(
...
+                    "#9467bd",  # muted purple
+                    "#8c564b",  # chestnut brown
+                    "#e377c2",  # raspberry yogurt pink
+                    "#7f7f7f",  # middle gray
+                    "#bcbd22",  # curry yellow-green
+                    "#17becf",  # blue-teal
+                ],
+            )
         )
-        self.plotly_theme = app.webviz_settings["theme"].plotly_theme
```
Comment (asnyv) [inline webviz_subsurface/plugins/_well_cross_section_fmu.py:138]: This change of behavior for colors should probably be in a separate PR, just something I realized when writing the docs. Anyways: seems like there is a bug that overwrites the input colors with theme colors if theme colors exist. Should rather use theme colors only if input colors are None.  Will revert this and write a separate issue before a merge.

Comment (asnyv) [reply webviz_subsurface/plugins/_well_cross_section_fmu.py:138]: Feels like I changed the style a bit while working through this. Especially when it comes to the "Data input" section. Not entirely sure what the best format is. 

Also a lot of notes, `code styled words`, code blocks and hyperlinks make it look a bit messy (especially since they are all colored). Not sure if clean, or "clear" (with e.g. `code styled words`) are the right way to go. Many colors and fonts make it a bit tiring to read it in my opinion.

**Comment by asnyv** [comment]
Feels like I changed the style a bit while working through this. Especially when it comes to the "Data input" section. Not entirely sure what the best format is. 

Also a lot of notes, `code styled words`, code blocks and hyperlinks make it look a bit messy (especially since they are all colored). Not sure if clean, or "clear" (with e.g. `code styled words`) are the right way to go. Many colors and fonts make it a bit tiring to read it in my opinion.

**Comment by asnyv** [comment]
Also `pylint` complains on wrong import order of third party module `yaml`, which is not happening locally even after updating `pylint` to `2.5.3` as in the CI run. Can't really see the issue as it is already grouped together with the third party imports.

File: webviz_subsurface/plugins/_well_cross_section_fmu.py (hunk 1 above)
Comment (asnyv) [reply webviz_subsurface/plugins/_well_cross_section_fmu.py]: This is moved to #382

**Comment by asnyv** [comment]
> Also `pylint` complains on wrong import order of third party module `yaml`, which is not happening locally even after updating `pylint` to `2.5.3` as in the CI run. Can't really see the issue as it is already grouped together with the third party imports. Will look more into it (or just change all of them to make `pylint` happy).

Fixed in #381

**Comment by asnyv** [comment]
Suggest to leave deploying documentation only at releases in a separate PR (equivalent to https://github.com/equinor/webviz-config/pull/247).

File: webviz_subsurface/plugins/_inplace_volumes_onebyone.py [hunk 2]
```diff
@@ -24,53 +24,72 @@
...
-The volumetric csv files must follow FMU standards.
-[Example csv file](
-https://github.com/equinor/webviz-subsurface-testdata/blob/master/aggregated_data/volumes.csv)
+* **`csvfile_vol`:** Aggregated csvfile for volumes with `REAL`, `ENSEMBLE` and `SOURCE` columns.
+* **`csvfile_parameters`:** Aggregated csvfile of parameters for sensitivity information with
+`REAL`, `ENSEMBLE`, `SENSNAME` and `SENSCASE` columns.
+* **`ensembles`:** Which ensembles in `shared_settings` to visualize (not to be used with
+`csvfile_vol` and `csvfile_parameters`).
+* **`volfiles`:**  Key/value pair of csv files when using `ensembles`.
+E.g. `{geogrid: geogrid--oil.csv}`.
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_inplace_volumes_onebyone.py:41]: Do we need `\` here?

File: webviz_subsurface/plugins/_relative_permeability.py [hunk 3]
```diff
@@ -16,21 +17,51 @@
...
+* **`sheet_name`:** Which sheet to use for the `scalfile`, only relevant if `scalfile` is an \
+    `xlsx` file (recommended to use csv files with `Webviz`).
+
+---
+The minimum requirement is to define `ensembles`.
+
+If no `relpermfile` is defined, the relative permeability data will be extracted automatically \
+from the simulation decks of individual realizations using `fmu-ensemble`and `ecl2df` behind the \
+scenes. Note that this method can be very slow for larger data decks, and is therefore not \
+recommended unless you have a very simple model/data deck.
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_relative_permeability.py:38]: Is `\` needed on text paragraphs?

File: webviz_subsurface/plugins/_relative_permeability.py [hunk 4]
```diff
@@ -16,21 +17,51 @@
...
+
+The file can e.g. be dumped to disc per realization by a forward model in ERT using \
+`ecl2df` or `pyscal`. \
+
+
+`scalfile` is a path to __a single file of SCAL recommendations__ (for all \
+realizations/ensembles). The file has to be compatible with \
+[pyscal's](https://equinor.github.io/pyscal/pyscal.html#pyscal.\
+factory.PyscalFactory.load_relperm_df) input format. Including this file adds reference cases \
+`Pess`, `Base` and `Opt` to the plots.
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_relative_permeability.py:56]: Same question.

File: webviz_subsurface/plugins/_reservoir_simulation_timeseries_regional.py [hunk 5]
```diff
@@ -38,44 +38,73 @@
...
- single file used for all ensembles_. If all region numbers for a filter are missing in the data,
- this filter value will be silently ignored. E.g. if no vectors matches 5 or 6 in
- [this example file](
- https://github.com/equinor/webviz-subsurface-testdata/tree/master/reek_history_match/share/
-regions/fip.yaml), `ZONE` == `LowerReek` would be ignored in the plugin for `FIPNUM`. This is
- to allow you to use the same file for e.g. a sector and a full field model.
-* `initial_vector`: First vector to plot (default is `ROIP`)
-* `column_keys`: List of vector patterns to include or `None`, the latter gives all available
- vectors, and `None` is also the default.<br/>
+    """Aggregates and visualizaes regional time series data from simulation ensembles. That
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_reservoir_simulation_timeseries_regional.py:41]: `visualizes`

File: webviz_subsurface/plugins/_reservoir_simulation_timeseries_regional.py [hunk 6]
```diff
@@ -38,44 +38,73 @@
...
+is: cumulatives, rates and inplace volumes. Allows human friendly filter names, e.g. regions,
+zones and etc based on user input.
+
+In addition recovery is calculated based on the changes in aggregated inplace volumes, \
+as long as all historical data is present in the data.
+
+Example of aggregation of ROIP over regions in filter: \
+$Agg(ROIP)_{date} = \\sum_{N\\in filter} {ROIP_N}_{date}$
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_reservoir_simulation_timeseries_regional.py:49]:
```suggestion
${\\rm Agg}({\\rm ROIP})_{\\rm date}} = \\sum_{N\\in \\rm filter}} {\\rm ROIP}_N}_{\\rm date}}$
```
Only a style suggestion to consider comitting to.... I'm used to roman style on variables that are spelled out with "text".

File: webviz_subsurface/plugins/_running_time_analysis_fmu.py [hunk 7]
```diff
@@ -15,25 +15,38 @@
...
-* Parallel coordinate plot for parameters used in realizations, colored by successful/failed
-realization or realization running time.
-
-* `ensembles`: Which ensembles in `shared_settings` to include in check. Only required input.
-* `filter_shorter`: Filters jobs with maximum run time in ensemble less than X seconds.
-Default: 10. Can be checked on/off interactively, this only sets the filtering value.
-* `status_file`: Name of json file with job status. Default: `status.json`
-* 'visual_parameters': List of default visualized parameteres in parallel coordinates plot.
-Default: all parameters.
+    """This plugin can e.g. be used to investigate which jobs that are important for the running \
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_running_time_analysis_fmu.py:18]: Remove `this plugin`?

File: webviz_subsurface/plugins/_subsurface_map.py [hunk 8]
```diff
@@ -13,20 +13,43 @@
 
 
 class SubsurfaceMap(WebvizPluginABC):
-    """### Subsurface map
-
-This plugin visualizes reservoir grids in a map view, additionally it can
+    """This plugin visualizes reservoir grids in a map view, additionally it can
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_subsurface_map.py:16]: Remove `this plugin`?

File: webviz_subsurface/plugins/_surface_viewer_fmu.py [hunk 9]
```diff
@@ -22,31 +22,61 @@
 
 
 class SurfaceViewerFMU(WebvizPluginABC):
-    """### SurfaceViewerFMU
+    """A plugin to covisualize surfaces from an ensemble.
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_surface_viewer_fmu.py:25]: Remove `a plugin`?

File: webviz_subsurface/plugins/_inplace_volumes_onebyone.py (hunk 2 above)
Comment (asnyv) [reply webviz_subsurface/plugins/_inplace_volumes_onebyone.py:41]: Not really, you need a whiteline for it to be read as a new line. Think I've rather overused than underused the \. On the other hand, it feels clearer that it is a part of the bullet point if adding a \, so think I will do that 👍

File: webviz_subsurface/plugins/_relative_permeability.py (hunk 3 above)
Comment (asnyv) [reply webviz_subsurface/plugins/_relative_permeability.py:38]: No, just a habit I aquired on the way as it is easier to (almost always) remember to do it, than to remember it when you have to...

File: webviz_subsurface/plugins/_relative_permeability.py (hunk 4 above)
Comment (asnyv) [reply webviz_subsurface/plugins/_relative_permeability.py:56]: same answer ;) But for the hyperlinks [somethink](a/link/somewhere) you have to

File: webviz_subsurface/plugins/_reservoir_simulation_timeseries_regional.py (hunk 6 above)
Comment (asnyv) [reply webviz_subsurface/plugins/_reservoir_simulation_timeseries_regional.py:49]: In that case I would suggest to keep it clean, drop roman and go sans serif (\\sf) as the rest of the text. Comparison of 
sans serif (upper equation) and roman (lower equation):
image
